include/http/libcurl_http_client.h

```diff
@@ -206,14 +228,27 @@ namespace azure {  namespace storage_lite {
         {
             REQUEST_TYPE *p = static_cast<REQUEST_TYPE *>(userdata);
             auto &s = p->m_input_stream.istream();
+            size_t streamlen = p->get_input_stream_length();
+            size_t actual_size = 0 ;
+            if( ! p->get_is_stream_length() ) {
```

Comment:
This code path will never hit because streamlen has a default value 0 and upload_block_blob_from_stream in src/blob/blob_client.cpp will always set_is_stream_length, right?